doc/references.bib

```diff
@@ -376,13 +376,23 @@ @article{FishburnEtAl2019
   publisher={Elsevier}
 }
 
-@article{GlasserEtAl2016,
+@article{GlasserEtAl2016a,
```

Comment:
instead of a/b, maybe better to leave this one and have `supp` for the other one?